### ISSUE TITLE
TalkBack reads Reader tab selected topic when launching in Me

### ISSUE THREAD

**Issue by astralbodies** [wordpress-mobile/WordPress-Android#6934]
### Expected behavior
When launching the app for the first time (after an update or a force-quit) the current tab should only be spoken by TalkBack.

### Actual behavior
The current tab was announced (Me - View Your Profile & Make Changes) but then the current selected topic in Reader was also announced after (in my case "Grand Meetup").

### Steps to reproduce the behavior
1. Select the Me tab.
2. Force quit an app.
3. Turn on TalkBack.
3. Relaunch the app.
4. Me should be announced and then the current topic is accidentally announced..
